test/e2e/petset.go

```diff
@@ -62,9 +62,7 @@ var _ = framework.KubeDescribe("PetSet", func() {
 		// dynamic volume provisioner.
 		framework.SkipUnlessProviderIs("gce")
 
-		var err error
-		c, err = framework.LoadClient()
-		Expect(err).NotTo(HaveOccurred())
+		c = f.Client
```

Comment:
This seems to be a common pattern in tests to assing client in BeforeEach. I wanted to be consistent with other places, e.g.:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/daemon_set.go#L85
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/kubectl.go#L179
...
(there are 19 such places)

If you think we should change that, can I send a separate PR to fix all of those in one, and for this one leave as is?